common/src/commonMain/kotlin/entity/DiscordMessage.kt

```diff
@@ -66,6 +66,7 @@
             "SuppressNotifications", shift = 12, kDoc = "This message will not trigger push and desktop notifications.",
         ),
         Entry("IsVoiceMessage", shift = 13, kDoc = "This message is a voice message."),
+        Entry("IsComponentsV2", shift = 15, kDoc = "Enable V2 of the Components API"),
```

Comment:
```suggestion
        Entry(
            "IsComponentsV2", shift = 15,
            kDoc = "allows you to create fully [component](https://discord.com/developers/docs/components/overview)-" +
                "driven messages.",
        ),
```
also `docUrl` is out of date, it should be https://discord.com/developers/docs/resources/message#message-object-message-flags